.travis.yml

```diff
@@ -1,44 +1,106 @@
-os: linux
-sudo: required
 language: python
```

Comment:
We may want to remove travis CI for easier maintenance since we use GH Actions now. But let's keep it for now just in case.